modules/ravestate_nlp/yes_no.py

```diff
@@ -0,0 +1,23 @@
+NEGATION_SET = {"neg"}
+
+
+def yes_no(doc):
+    """
+    checks input for "yes", "no", "i don't know", "probably" and "probably not"
+    """
+    nlp_tokens = tuple(str(token) for token in doc)
+    for token in nlp_tokens:
+        if token == "yes" or token == "y":
+            return "yes"
+        elif token == "no" or token == "n":
```

Comment:
maybe check case insensitive
But that might be a change to do somewhere higher up the nlp chain :wink:

Reply:
Done :) Thanks